doc/cephfs/add-remove-mds.rst

```diff
@@ -107,3 +113,5 @@ the following method.
 	$ sudo rm -rf /var/lib/ceph/mds/ceph-${id}
 
 .. _MDS Config Reference: ../mds-config-ref
+.. _cephadm: ../../cephadm/index.html
+.. _ceph orchestrator: ../../cephadm/services/mds/
```

Comment:
Here also: s/ceph/Ceph

Reply:
drop this change and use `:ref:` instead.

Reply:
Yeah,that would be better:)

Reply:
To help you, read: https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html#cross-referencing-using-roles. `:doc:` could also be used.

Reply:
I've made the changes